src/signals/incident-management/containers/IncidentDetail/index.test.js

```diff
@@ -1,10 +1,399 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
 
-// import IncidentDetail from 'index';
+import { REQUEST_INCIDENT, PATCH_INCIDENT, REQUEST_ATTACHMENTS, REQUEST_DEFAULT_TEXTS, DISMISS_SPLIT_NOTIFICATION, DISMISS_ERROR } from 'models/incident/constants';
+import { REQUEST_HISTORY_LIST } from 'models/history/constants';
+import LoadingIndicator from 'shared/components/LoadingIndicator';
+
+import { IncidentDetail, mapDispatchToProps } from './index';
+
+import DetailHeader from './components/DetailHeader';
+import MetaList from './components/MetaList';
+import History from './components/History';
+import AddNote from './components/AddNote';
+import LocationForm from './components/LocationForm';
+import AttachmentViewer from './components/AttachmentViewer';
+import StatusForm from './components/StatusForm';
+import Detail from './components/Detail';
+import SplitNotificationBar from './components/SplitNotificationBar';
+import LocationPreview from './components/LocationPreview';
+
+import statusList, { changeStatusOptionList } from '../../definitions/statusList';
+import stadsdeelList from '../../definitions/stadsdeelList';
+import priorityList from '../../definitions/priorityList';
 
 describe('<IncidentDetail />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  let wrapper;
+  let instance;
+  const props = {
+    id: '42',
```

Comment:
If you put this in a fixtures file, the test suite will be compacte and easier to consume (by humans).